app/models/work_view_stat.rb

```diff
@@ -0,0 +1,19 @@
+class WorkViewStat < ActiveRecord::Base
+  extend Sufia::WorkStatUtils
```

Comment:
Why are you putting these methods in a module?  It doesn't appear that the module would be able to be used in any other context.